content/22.vaccines-trad.md

```diff
@@ -659,7 +659,7 @@ Overall, the vaccine landscape remains heterogeneous even as the pandemic nears
 The vaccines described in this manuscript, which were developed using well-established technologies, have played a crucial role in improving the feasibility and accessibility of vaccination programs worldwide.
 While the novel technologies have received the bulk of public attention in countries like the U.S., these more traditional vaccine platforms also provide safe and highly effective protection against SARS-CoV-2.
 Although companies developing cutting-edge technologies, namely Moderna and Pfizer/BioNTech, reported very high efficacies greater than 90% in their clinical trials [@individual-vaccines-novel], the efficacies identified in whole-virus and subunit trials have also been very high.
-While the clinical trial efficacy estimates for IWV and subunit have been lower, some of these trials have also reported efficacies over 80% (e.g., Novavax's Nuvaxovid, 89% [cite]; CoronaVac 83.5% [cite]).
+While the clinical trial efficacy estimates for IWV and subunit have been lower, some of these trials have also reported efficacies over 80% (e.g., Novavax's Nuvaxovid, 89% [cite]; CoronaVac 83.5% [cite]). <!-- missing two citations here -->
```

Comment:
I'm not sure what these citations should be

Reply:
```suggestion
While the clinical trial efficacy estimates for IWV and subunit have been lower, some of these trials have also reported efficacies over 80% (e.g., Novavax's Nuvaxovid with 89.7% [@doi:10.1056/NEJMoa2107659] or Sinovac's CoronaVac with 83.5% [@doi:10/gk898z]).
```
Oops, thank you for catching that I forgot to go back and fill these in!